mqtt/mqttd/src/main.rs

```diff
@@ -120,3 +109,19 @@ async fn tick_snapshot(
         }
     }
 }
+
+#[allow(deprecated)]
```

Comment:
There is another macro `app_from_crate` that does what we need. And doesn't contain deprecated code!
```rust 
fn create_app() -> App<'static, 'static> {
    app_from_crate!().arg(
        Arg::with_name("config")
            .short("c")
            .long("config")
            .value_name("FILE")
            .help("Sets a custom config file")
            .takes_value(true),
    )
}
```

Reply:
it is the create_authors! which deprecated. I can remove that, doesn't hold much information anyway.